modules/tinymce/src/core/main/ts/content/GetContentImpl.ts

```diff
@@ -15,52 +15,91 @@ import Tools from '../api/util/Tools';
 import { isWsPreserveElement } from '../dom/ElementType';
 import * as TrimHtml from '../dom/TrimHtml';
 import * as Zwsp from '../text/Zwsp';
-import { Content, ContentFormat, GetContentArgs } from './ContentTypes';
+import { Content, GetContentArgs } from './ContentTypes';
+import { GetContentFormatter } from './EditorContent';
+
+const defaultContentFormatter = (editor: Editor, args: GetContentArgs): Content =>
+  Optional.from(editor.getBody())
+    .fold(
+      Fun.constant(args.format === 'tree' ? new AstNode('body', 11) : ''),
+      (body) => getContentFromBody(editor, args, body)
+    );
+
+const addDefaultGetFormats = (formatCell: Cell<Record<string, GetContentFormatter>>) => {
```

Comment:
This inline type is getting quite verbose and is repeated a few times, could we move it out?

Reply:
Something like `GetContentRegistry` and `SetContentRegistry` might be a good name for this :shrug: